verticapy/tests/sql/test_sql.py

```diff
@@ -167,6 +168,35 @@ def test_sql(self, titanic_vd):
         os.remove("verticapy_test_sql.csv")
         file.close()
 
+        # Testing the replace_external_queries_in_query function
+        set_external_connection("my_external_cid")
+
+        assert (
+            get_dblink_fun("  My TEsT QUERY  ")
+            == "SELECT DBLINK(USING PARAMETERS cid='my_external_cid', query='  My TEsT QUERY  ', rowset=500) OVER ()"
```

Comment:
usually in these queries, they end by " ; ". I do not see it here.

Reply:
We do not necessarily end the query. This code can belong to a subquery.